test/e2e/framework/framework.go

```diff
@@ -95,6 +96,17 @@ type Framework struct {
 	// Federation specific params. These are set only if federated = true.
 	FederationClientset_1_5 *federation_release_1_5.Clientset
 	FederationNamespace     *v1.Namespace
+
+	// CRI client
+	CRIClient *InternalApiClient
```

Comment:
Why you need e2e framework for cri validation test? I think we should add another util or framework.